### DISCUSSION TITLE
Creating a website with Jupyter Book has a different layout than JupyterLab-MyST

### DISCUSSION THREAD

**Discussion by sbonaretti** [executablebooks/meta#1199] [Q&A] [answered]
Hey guys, 

Thanks a lot for creating these fantastic tools! I am in the process of learning some new tools, and I have a quick question (hoping that this is the right place!)

I am creating a website for a package documentation using JupyterLab-MyST in Jupyter Notebook. I used the fantastic callouts that MyST provides and that are super cool! But when I create the website using the JupyterBook built, the callouts change their layout. 

<img width="1049" alt="Screenshot 2024-07-22 at 18 13 21" src="https://github.com/user-attachments/assets/6efc48b5-20d0-4d0f-88ff-24d04312f0c9">

I have tried to do the manual installation of NodeJs and create the website from there, but it is a bit too complicated. This website wants to be a collaborative projects, and we need to keep the computational complexity at minimum. 

My question is: *Is there are way to create the website using JupyterLab-MyST using JupyterBook and keep the same layout?*

Thanks a lot for all your hard work!!!
Serena

**Comment by choldgraf** [comment, marked as answer]
This is because Jupyter Book 1.0 uses a different document engine for MyST Markdown than what JupyterLab uses.

Jupyter Book uses [Sphinx](https://sphinxdoc.org) along with [the MyST Markdown Parser](https://myst-parser.readthedocs.io) and [the Sphinx Book Theme](https://sphinx-book-theme.readthedocs.io).

JupyterLab MyST uses [the new MyST document engine](https://mystmd.org) which will one day (soon hopefully) replace Sphinx as the main document engine of Jupyter Book. You can use [mystmd](https://mystmd.org) on its own right now if you wish!

Some more context is in [this blog post about Jupyter Book 2.0 and MyST](https://executablebooks.org/en/latest/blog/2024-05-20-jupyter-book-myst/).

**Reply by sbonaretti** [reply]
Got it! Thanks @choldgraf !!!

**Reply by rowanc1** [reply]
For compatibility you can use the "admonition" directive with a class, which takes a custom title in both cases:

![image](https://github.com/user-attachments/assets/7516dc52-5b2d-4426-a51b-5eb4b003f49b)

https://mystmd.org/guide/admonitions

**Reply by sbonaretti** [reply]
Fantastic, it works!!! Thanks!!! :)
